source/includes/fundamentals/code-examples/configure/ConfigureEFProvider.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.EntityFrameworkCore;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using Microsoft.Extensions.Configuration;
+using MongoDB.EntityFrameworkCore.Extensions;
+
+var mongoClient = new MongoClient("<Your MongoDB Connection URI>");
+var mongoDatabase = mongoClient.GetDatabase("<Database Name>");
+
+var dbContextOptions =
+    new DbContextOptionsBuilder<MyDbContext>().UseMongoDB(mongoClient,
+        mongoDatabase.DatabaseNamespace.DatabaseName);
```

Comment:
Would probably be simpler to change this line to say `"<Database Name">` and eliminate line 8 entirely.

I know the example on the GitHub README uses `GetDatabase` but that was so we could pass the client and database name in together which isn't necessary here.